tracer/src/Datadog.Trace/Ci/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -216,9 +216,9 @@ private int WriteTags(ref byte[] bytes, int offset, Span span, ITags tags, ITagP
             offset = tagWriter.Offset;
             count += tagWriter.Count;
 
-            // TODO: for each trace tag, determine if it should be added to the local root,
-            // to the first span in the chunk, or to all orphan spans.
-            // For now, we add them to the local root which is correct in most cases.
+            // Write trace tags
+            // NOTE: this formatter for CI Visibility doesn't know if the span is "first in chunk" or "chunk orphan",
```

Comment:
@tonyredondo do we need to update this formatter to support 128-bit trace ids (`_dd.p.tid`)?